<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />

    <title></title>
    <link rel="stylesheet" href="dist/reveal.css" />
    <link rel="stylesheet" href="dist/theme/white.css" id="theme" />
    <link rel="stylesheet" href="plugin/highlight/zenburn.css" />
	<link rel="stylesheet" href="css/layout.css" />
	<link rel="stylesheet" href="plugin/customcontrols/style.css">
	<link rel="stylesheet" href="plugin/chalkboard/style.css">

	<link rel="stylesheet" href="plugin/reveal-pointer/pointer.css" />


    <script defer src="dist/fontawesome/all.min.js"></script>

	<script type="text/javascript">
		var forgetPop = true;
		function onPopState(event) {
			if(forgetPop){
				forgetPop = false;
			} else {
				parent.postMessage(event.target.location.href, "app://obsidian.md");
			}
        }
		window.onpopstate = onPopState;
		window.onmessage = event => {
			if(event.data == "reload"){
				window.document.location.reload();
			}
			forgetPop = true;
		}

		function fitElements(){
			const itemsToFit = document.getElementsByClassName('fitText');
			for (const item in itemsToFit) {
				if (Object.hasOwnProperty.call(itemsToFit, item)) {
					var element = itemsToFit[item];
					fitElement(element,1, 1000);
					element.classList.remove('fitText');
				}
			}
		}

		function fitElement(element, start, end){

			let size = (end + start) / 2;
			element.style.fontSize = `${size}px`;

			if(Math.abs(start - end) < 1){
				while(element.scrollHeight > element.offsetHeight){
					size--;
					element.style.fontSize = `${size}px`;
				}
				return;
			}

			if(element.scrollHeight > element.offsetHeight){
				fitElement(element, start, size);
			} else {
				fitElement(element, size, end);
			}		
		}


		document.onreadystatechange = () => {
			fitElements();
			if (document.readyState === 'complete') {
				if (window.location.href.indexOf("?export") != -1){
					parent.postMessage(event.target.location.href, "app://obsidian.md");
				}
				if (window.location.href.indexOf("print-pdf") != -1){
					let stateCheck = setInterval(() => {
						clearInterval(stateCheck);
						window.print();
					}, 250);
				}
			}
	};


        </script>
  </head>
  <body>
    <div class="reveal">
      <div class="slides"><section ><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# It is the year 20XX
</div>

<aside class="notes"><p>I want to borrow your imagination for but a moment. Imagine the future, your future that is.</p>
</aside></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<p class="fragment"><i>You sit in your office, the crumbling headquarters of the resistance against the AI.</i></p>

<p class="fragment"><i>Explosions and gunfire can be heard through the barricaded doors.</i></p>

<p class="fragment">"Boss, we can't hold for much longer. What are your orders?"</p>
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<p class="fragment"><i>You hesitate for only the briefest of moments.</i></p>

<p class="fragment"><b>"Activate the time machine. Send the presentation files back to 2023-"</b></p>

<p class="fragment"><i>The barricade breaks. The final dance begins. You can only hope you have done enough.</i></p>
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<h3 class ="fragment">Transcriptomics and more for long-lived nematode mutants</h3>
<h5 class ="fragment">Presented by</h5>
<h3 class ="fragment">Brandon Penny</h3>
</div>

<aside class="notes"><p>Unrelated presentations actually. I don&#39;t know where that other one got sent to.</p>
</aside></script></section></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-transition-speed="slow" -->

### Where can we see elegance?
</div>

<aside class="notes"><p>I know after this presentation you will all have too many questions for me. But for now I will ask all of you a simple question. &quot;Where can we see elegance?&quot; ... How did you guys know it was a rhetorical question?</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition-speed="slow" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-transition="convex-in slide-out" -->

<h1 class>C. Elegans</h1>
</div>

<aside class="notes"><p>Yes, we can see elegance in C. Elegans.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="Core/Assets/Attachments/Pasted image 20230403151741.png" alt="" style="width: 900px; height: 600px; object-fit: fill">

<p>
Fig. 1: Anatomical diagram of an adult-stage C. elegans hermaphrodite </p>
</div>

<aside class="notes"><p>Here it is. The model nematode. Easy to breed, relatively simple, and has a compact life-cycle.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

> How genes might specify the complex structures found in higher organisms is a major unsolved problem in biology.
> 
\- Dr. Sydney Brenner, 1973
</div>

<aside class="notes"><p>...some of the main reasons it was proposed as a model organisms. By understanding these creatures we can understand ourselves.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>
<div data-auto-animate-easing = "ease-in-out">
<div data-id="box1" auto-animate-delay="0" style="background: gold; width: 200px; height: 150px; border-radius: 15px 15px; margin: 10px; display: flex; justify-content: center; align-items: center; font-family: Montserrat">G</div>
</div>

<aside class="notes"><p>Studying homology between humans and these model organisms is a popular endeavor. But the extent of a homologous gene&#39;s influence can be hard to measure.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>
<div data-auto-animate-easing = "ease-in-out">
<h2 style="text-align: right-justify;">Pathways</h2>

<split>
<div data-id="box1" auto-animate-delay="0" style="background: gold; width: 200px; height: 150px; border-radius: 15px 50px; margin: 10px; display: flex; justify-content: center; align-items: center; font-family: Montserrat">G1</div>


<div data-id="box2" auto-animate-delay="0.1" style="background: darkorange; width: 200px; height: 150px; border-radius: 15px 50px; margin: 10px; display: flex; justify-content: center; align-items: center; font-family: Montserrat">G2</div>


<div data-id="box3" auto-animate-delay="0.2" style="background: orangered; width: 200px; height: 150px; border-radius: 15px 50px; margin: 10px; display: flex; justify-content: center; align-items: center; font-family: Montserrat">G3</div>

</split>
</div>

<aside class="notes"><p>Sometimes it is better to focus on specific functional pathways. </p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>
<div data-auto-animate-easing = "ease-in">
<div class="r-stack">
<div data-id="box1" style="background: gold; width: 350px; height: 350px; border-radius: 200px; font-family: Montserrat">G1</div>


<div data-id="box2" style="background: darkorange; width: 250px; height: 250px; border-radius: 200px; font-family: Montserrat">G2</div>


<div data-id="box3" style="background: orangered; width: 150px; height: 150px; border-radius: 200px; display: flex; justify-content: center; align-items: center; font-family: Montserrat">G3</div>
</div>
</div>

<aside class="notes"><p>Like signaling pathways, starting from the exterior of a cell.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# The Dao of Aging
</div>

<aside class="notes"><p>The pathways we are primarily interested in are the ones that influence aging and longevity. There are many ways we can age so lets just pick a path.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in fade-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-transition-speed="slow" -->

<img src="Core/Assets/Attachments/Pasted image 20230403185218.png" alt="" style="width: 900px; height: 600px; object-fit: fill">

Fig. 2: Insulin/IGF-1 signaling pathway (IIS)
</div>

<aside class="notes"><p>Decently conserved. The IIS pathway is connected to several functions but aging is the coolest one. There are many different potential targets here that have been found to modulate lifespan. But we have one specific target in mind.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="zoom-in fade-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-transition-speed="slow" -->

<img src="Core/Assets/Attachments/Pasted image 20230404231540 1.png" alt="" style="width: 900px; height: 600px; object-fit: fill">
</div>

<aside class="notes"><p>PI3K, a kinase, and the gene behind the scenes, age-1.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="zoom-in" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

# AGE-1

## (Canonical)

- p85 binding
- Ras binding
- C2 domain
- Helical domain
- Kinase domain

<div>
</div>

<aside class="notes"><p>Normally a critical link in the phosphorylating cascade. But what happens when you take the business end away?</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

# AGE-1 

## (MG44 allele)

- p85 binding
- Ras binding
- C2 domain
- STOP <!-- .element: style="color: red" -->
- ~~Helical domain~~
- ~~Kinase domain~~

<div>
</div>

<aside class="notes"><p>Caused from a mutation inserting a stop codon resulting in a truncated protein. Now where does that leave us?</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in concave-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

A dead end

<div data-id="mom">
<i class="fas fa-female fa-2x"></i>
</div>
<br>
<div data-id="2nd">
<i class="fas fa-skull fa-1x"></i>
</div>
Here lay the second generation

<div>
</div>

<aside class="notes"><p>Nowhere, the homozygotes with these alleles live a little longer but are completely infertile, their offspring never growing into adults.</p>
</div></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="concave-in" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

<div data-id="status">?</div>

<div data-id="mom">
<i class="fas fa-female fa-2x"></i>
</div>
<br>
<br>
<div data-id="2nd">
<i class="fas fa-egg fa-1x"></i>
</div>

<div>
</div>

<aside class="notes"><p>But wait...</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

<div data-id="status">!!!</div>

<div data-id="mom">
<i class="fas fa-female fa-2x"></i>
</div>
<br>
<br>
<br>
<div data-id="2nd">
<i class="fas fa-male fa-1x"></i>
</div>

<div>
</div>

<aside class="notes"><p>What is this?</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-auto-animate="true" -->

<div data-id="2nd">
<i class="fas fa-male fa-3x"></i>
</div>

<p class="fragment">It lives</p>

<div>
</div>

<aside class="notes"><p>Under certain chill conditions, (lowered temperature and longer time before tossing the sample) the 2nd generation of these mutants actually becomes adults.</p>
</div></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in slide-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Germ of infirm worms confirmed to squirm
</div>

<aside class="notes"><p>A possible headline</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="Core/Assets/Attachments/Pasted image 20230403194015.png" alt="" style="width: 900px; height: 600px; object-fit: fill">

Fig. 3: Survivals of F2 age-1 (mg44) homozygotes
</div>

<aside class="notes"><p>Among worms, a veritable Methuselah. Up to 10x longer lifespan than the wild-type.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<h3>The process</h3>

<p class="fragment">RNA-Seq </p>

<p class="fragment">Quality control</p>

<p class="fragment">Mapping</p>

<p class="fragment">Counting</p>

<p class="fragment">Differential expression</p>
</div>

<aside class="notes"><p>So what can be done? We have this long lived mutant. But why is it living this long? It has the same DNA as its parents so how do we determine the cause of this incredible lifespan? Transcriptomics. What genes are being expressed in these mutants differently compared their parents or the wild type nematodes?</p>
<p>So how do we do this?</p>
<p>...</p>
<p>But first lets take a step back.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### DNA

<section data-auto-animate>
<div data-auto-animate-easing = "ease-in-out">
<split>
<div data-id="exon1" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="intron1" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="exon2" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="intron2" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 0px; height: 0px;  border-top: 25px solid white; border-left: 50px solid orange; border-bottom: 25px solid white;"></div>
</split>

<br>

<split>
<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 0px; height: 0px;  border-top: 25px solid white; border-right: 50px solid orange; border-bottom: 25px solid white;"></div>

<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="intron1" auto-animate-delay="0" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="exon2" auto-animate-delay="0" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="intron2" auto-animate-delay="0" style="background: orange; width: 100px; height: 50px;"></div>

<div data-id="exon3" auto-animate-delay="0" style="background: orange; width: 100px; height: 50px;"></div>

</split>
</div>

<div>
</div>

<aside class="notes"><p>Assume we have some DNA.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>
<div style = "font-family: Montserrat;">
<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orangered; width: 100px; height: 50px; margin: 10px;">E1</div>

<div data-id="intron1" auto-animate-delay="0" style="background: gold; width: 100px; height: 50px; margin: 10px;">I1</div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orangered; width: 300px; height: 50px; margin: 10px;">E2</div>

<div data-id="intron2" auto-animate-delay="0" style="background: gold; width: 100px; height: 50px; margin: 10px;">I2</div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orangered; width: 200px; height: 50px; margin: 10px;">E3</div>

</split>
</div>

<div>
</div>

<aside class="notes"><p>Eukaryotic genes are composed of exons and introns</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

### RNA

<div style = "font-family: Montserrat;">
<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orangered; width: 100px; height: 50px; margin: 10px;">E1</div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orangered; width: 300px; height: 50px; margin: 10px;">E2</div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orangered; width: 200px; height: 50px; margin: 10px;">E3</div>

</split>

</div>

<div>
</div>

<aside class="notes"><p>We want info about the various transcripts that can be spliced together from these genes.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

### Sequence

<split>

<div data-id="ad1" auto-animate-delay="0" style="background: steelblue; width: 50px; height: 50px;"></div>

<div data-id="exon1" auto-animate-delay="0" style="background: orangered; width: 200px; height: 50px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orangered; width: 200px; height: 50px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orangered; width: 200px; height: 50px;"></div>

<div data-id="ad2" auto-animate-delay="0" style="background: steelblue; width: 50px; height: 50px;"></div>

</split>


<br>

<split even>

<div data-id="left">
<i class="fas fa-spinner fa-2x fa-pulse"></i>
</div>
<div data-id="middle">
<i class="fas fa-laptop-code fa-2x">
</i></div>
<div data-id="right">
<i class="fas fa-spinner fa-2x fa-pulse">
</i></div>
</split>

<div>
</div>

<aside class="notes"><p>We can black-box the sequencing process as long we don&#39;t forget about the specific biases from technical artifacts. </p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

### Reads

<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>
<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 150px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 75px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

</split>

<div>
</div>

<aside class="notes"><p>Behind the curtains of Oz our sequences got fragmented and those are used to generate short reads.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>
<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 150px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 75px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

</split>

<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 50px; margin-left: 60px; margin-top: 10px; margin-right: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 150px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 75px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

</split>

<split>


<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 150px; height: 50px; margin-left: 20px; margin-top: 10px; margin-right: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 50px; height: 50px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 50px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 150px; height: 50px; margin: 10px; padding-right: 100px;"></div>

</split>

<div>
</div>

<aside class="notes"><p>Lots of reads. Lots of data.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

## Mapping

<div data-id="refer" auto-animate-delay=".3" style="background: orangered; width: 800px; height: 50px; border: solid gold;"></div>

<split>

<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 10px; margin: 10px;"></div>
<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 10px; margin: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 150px; height: 10px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 50px; height: 10px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 75px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>

</split>

<split>


<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 50px; height: 10px; margin-left: 60px; margin-top: 10px; margin-right: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 150px; height: 10px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 75px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>

</split>

<split>


<div data-id="exon1" auto-animate-delay="0" style="background: orange; width: 150px; height: 10px; margin-left: 20px; margin-top: 10px; margin-right: 10px;"></div>

<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 50px; height: 10px; margin: 10px;"></div>
<div data-id="exon2" auto-animate-delay="0.1" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>

<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 25px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 100px; height: 10px; margin: 10px;"></div>
<div data-id="exon3" auto-animate-delay="0.2" style="background: orange; width: 150px; height: 10px; margin: 10px; padding-right: 100px;"></div>

</split>

<div>
</div>

<aside class="notes"><p>We then align these short read fragments to a reference genome and count the hits.</p>
</div>

</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">
<canvas style="max-height:700px" data-chart="bar" >
<!--
{"data":{"datasets":[{"data":[4,10],"label":"Wild Type","backgroundColor":"#4285f4"},{"data":[5,6],"label":"Mutant","backgroundColor":"#ea4335"}],"labels":["Gene A","Gene B"]},"options":{"elements":{"bar":{}}}}-->
</canvas>
</div>

<aside class="notes"><p>So we can then finally examine the differences in gene expression between the mutant and the wild type nematodes.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="has-light-background drop" data-background-color="darkgrey" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# Behind the scenes
</div>

<aside class="notes"><p>But in reality it is ugly work.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="has-light-background drop" data-background-color="silver" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-auto-animate="true" -->

<pre data-id="code-animation">
<code data-line-numbers="1-2|4-5|7|9">go to work folder
check quality of reads

trim low quality reads
check quality again

align reads to the reference genome

count mapped reads
</code>
</pre>
</div>

<aside class="notes"><p>This pseudocode details the journey of raw sequenced data into mapped transcript data.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="has-light-background drop" data-background-color="silver" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<!-- .slide: data-auto-animate="true" -->
<!-- .slide: data-transition="convex-out" -->

<pre data-id="code-animation">
<code data-line-numbers="1-2|4-6|8-9|11-12">cd rnaseq_directory
fastqc -f d/RawFiles.fastq -o res/

trimmomatic -f d/RawFiles.fastq 
	-o d/TrimmedFiles.fastq	
fastqc -f d/TrimmedFiles.fastq -o res/

hisat2 -f d/TrimmedFiles.fastq
	-t rna -r d/ref_genome
	
featurecounts -a d/annotated_genome.gtf 
	-o d/featurecounts.txt -f d/trimmed.bam
</code>
</pre>
</div>

<aside class="notes"><p>Here we see a more realistic depiction of the pipeline. FASTQC gives descriptive statistics, Trimmomatic removes low quality reads. Hisat2 aligns the reads to the genome. And featureCounts counts the features.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in slide-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

Transcript | Wild Type Control | Mutant Sample
------------ | ------------ |  ------- 
T001 | 42 | 13
T002 | 36 | 33
... | ... | ...
T998 | 52 | 47
T999 | 23 | 67
</div>

<aside class="notes"><p>So all that will give us a count matrix that looks something like this.  How can we determine what is actually being differentially expressed? That is the tricky part. Just because there are more counts for one transcript doesn&#39;t mean it is being expressed more, many biases might have to be accounted for.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Differential analysis

<p class="fragment">DESeq2 </p>

<p class="fragment">edgeR</p>

<p class="fragment">Wilcoxon rank-sum test</p>
</div>

<aside class="notes"><p>Most commonly used are parametric methods which may utilize their own normalization schemes like DESeq2.</p>
<p>These R packages are pretty standard and have been around for a bit.</p>
<p>Nonparametric methods can also be used and the scale matters less when you are more focused on the rank order.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# Applications
</div>

<aside class="notes"><p>ok ok we actually have to justify this process to the suits</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="Core/Assets/Attachments/Pasted image 20230413123513.png" alt="" style="width: 720px; height: 480px; object-fit: fill">


Fig 4: Summary of survivals for pharmacological interventions identified by CMAP.
</div>

<aside class="notes"><p>After you identify the differentially expressed genes you can compare the &quot;signature&quot; (what genes are up, what genes are down) against drug databases to look for potential lifespan-extending drugs. Although recent research has shown some promising leads, no single drug shows as drastic effects as the the mutants do by themselves.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in slide-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# What lies beyond
</div>

<aside class="notes"><p>What does the future hold? What problems need to be addressed for extending this research?</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

## Too expressive
</div>

<aside class="notes"><p>Your differentially expressed genes you found, are they actually being differentially expressed? Where could our analysis falter?</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

### Parametric

<split>

<div data-id="c1" auto-animate-delay="0.1" style="border-bottom: 200px solid  orangered; width: 50px; height: 0px; margin: 5px;"></div>

<div data-id="c2" auto-animate-delay="0.2" style="border-bottom: 125px solid  orangered; width: 50px; height: 75; margin: 5px;"></div>

<div data-id="c3" auto-animate-delay="0.3" style="border-bottom: 75px solid  orangered; width: 50px; height: 125px; margin: 5px;"></div>

<div data-id="c4" auto-animate-delay="0.4" style="border-bottom: 60px solid  orangered; width: 50px; height: 140px; margin: 5px;"></div>

<div data-id="c5" auto-animate-delay="0.5" style="border-bottom: 50px solid  orangered; width: 50px; height: 150px; margin: 5px;"></div>

<div data-id="c6" auto-animate-delay="0.6" style="border-bottom: 45px solid  orangered; width: 50px; height: 155px; margin: 5px;"></div>

<div data-id="c7" auto-animate-delay="0.7" style="border-bottom: 40px solid  orangered; width: 50px; height: 160px; margin: 5px;"></div>

<div data-id="c8" auto-animate-delay="0.8" style="border-bottom: 30px solid  orangered; width: 50px; height: 170px; margin: 5px;"></div>

</split>
<div>
</div>

<aside class="notes"><p>DESeq2 and edgeR assume a negative binomial distribution of the count data.</p>
</div>
</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<section data-auto-animate>

### Nonparametric

<split>

<div data-id="c1" auto-animate-delay="0.1" style="border-bottom: 150px solid  orangered; width: 50px; height: 50px; margin: 5px;"></div>

<div data-id="c2" auto-animate-delay="0.2" style="border-bottom: 150px solid  orangered; width: 50px; height: 50; margin: 5px;"></div>

<div data-id="c3" auto-animate-delay="0.3" style="border-bottom: 75px solid  orangered; width: 50px; height: 125px; margin: 5px;"></div>

<div data-id="c4" auto-animate-delay="0.4" style="border-bottom: 120px solid  orangered; width: 50px; height: 80px; margin: 5px;"></div>

<div data-id="c5" auto-animate-delay="0.5" style="border-bottom: 140px solid  orangered; width: 50px; height: 60px; margin: 5px;"></div>

<div data-id="c6" auto-animate-delay="0.6" style="border-bottom: 50px solid  orangered; width: 50px; height: 150px; margin: 5px;"></div>

<div data-id="c7" auto-animate-delay="0.7" style="border-bottom: 75px solid  orangered; width: 50px; height: 125px; margin: 5px;"></div>

<div data-id="c8" auto-animate-delay="0.8" style="border-bottom: 100px solid  orangered; width: 50px; height: 100px; margin: 5px;"></div>

</split>
<div>
</div>

<aside class="notes"><p>But what if that assumption is wrong? Could that lead to finding too many false positives among differentially expressed genes?</p>
</div>
</section></aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="convex-in slide-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Version control


<div class="mermaid">
gitGraph
   commit
   commit
   branch add-non-parametric
   checkout add-non-parametric
   commit
   commit
   checkout main
   merge add-non-parametric
   commit
   commit

</div>
</div>

<aside class="notes"><p>If there is code and if that code is likely to change then you should probably be using something like Git. Creating &quot;branches&quot; off your main &quot;trunk&quot; when you add some experimental feature to your workflow will ensure you don&#39;t permanently break your pipeline.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" data-transition="slide-in convex-out" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Reproducibility

<img src="Core/Assets/Attachments/Pasted image 20230411141228.png" alt="" style="object-fit: scale-down">
</div>

<aside class="notes"><p>These bioinformatics pipelines can require a couple dozen specific Python libraries, R packages, and various other system level programs. Sharing your data and scripts usually isn&#39;t that easy.</p>
</aside></script></section><section ><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="Core/Assets/Attachments/Pasted image 20230413202552.png" alt="" style="object-fit: scale-down">
</div>

<aside class="notes"><p>Containerization tech like Docker lets you &quot;contain&quot; your working environment and distribute it easily. So if it works on your computer, you can essentially just send your computer to someone else.</p>
<p>But we might need to actually drop this option.</p>
</aside></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

<img src="Core/Assets/Attachments/Pasted image 20230413202602.png" alt="" style="object-fit: scale-down">
</div>

<aside class="notes"><p>Although with how the permissions are configured for Docker, requiring root access, you may have to use alternatives like Singularity/Apptainer for running containers on a high performance cluster.</p>
</aside></script></section></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Chat bots

<img src="Core/Assets/Attachments/Pasted image 20230411134654.png" alt="" style="object-fit: scale-down">
</div>

<aside class="notes"><p>Potentially useful tools that can save you time in otherwise tedious tasks but these Large-Language-Models are still just mimics. Little value in the wild-west of biology.</p>
</aside></script></section><section  data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

### Transcriptomics and more for long-lived nematode mutants

###### Presented by

#### Bra(N)don Penny
</div></script></section><section ><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

# References
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

Fig. 1: K. D. Schroeder, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=26958836

Fig. 2: Murphy CT, Hu PJ. Insulin/insulin-like growth factor signaling in C. elegans. _WormBook_. Published online December 26, 2013:1-43. doi:[10.1895/wormbook.1.164.1](https://doi.org/10.1895/wormbook.1.164.1)
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

Fig. 3: Shmookler Reis RJ, Bharill P, Tazearslan C, Ayyadevara S. Extreme-longevity mutations orchestrate silencing of multiple signaling pathways. _Biochimica et Biophysica Acta (BBA) - General Subjects_. 2009;1790(10):1075-1083. doi:[10.1016/j.bbagen.2009.05.011](https://doi.org/10.1016/j.bbagen.2009.05.011)
</div></script></section><section data-markdown><script type="text/template"><!-- .slide: class="drop" -->
<div class="" style="position: absolute; left: 0px; top: 0px; height: 700px; width: 960px; min-height: 700px; display: flex; flex-direction: column; align-items: center; justify-content: center" absolute="true">

Fig. 4: Tarkhov AE, Alla R, Ayyadevara S, et al. A universal transcriptomic signature of age reveals the temporal scaling of Caenorhabditis elegans aging trajectories. _Sci Rep_. 2019;9:7368. doi:[10.1038/s41598-019-43075-z](https://doi.org/10.1038/s41598-019-43075-z)
</div></script></section></section></div>
    </div>

    <script src="dist/reveal.js"></script>

    <script src="plugin/markdown/markdown.js"></script>
    <script src="plugin/highlight/highlight.js"></script>
    <script src="plugin/zoom/zoom.js"></script>
    <script src="plugin/notes/notes.js"></script>
    <script src="plugin/math/math.js"></script>
	<script src="plugin/mermaid/mermaid.js"></script>
	<script src="plugin/chart/chart.min.js"></script>
	<script src="plugin/chart/plugin.js"></script>
	<script src="plugin/customcontrols/plugin.js"></script>
	<script src="plugin/chalkboard/plugin.js"></script>
	<script src="plugin/reveal-pointer/pointer.js"></script>

    <script>
      function extend() {
        var target = {};
        for (var i = 0; i < arguments.length; i++) {
          var source = arguments[i];
          for (var key in source) {
            if (source.hasOwnProperty(key)) {
              target[key] = source[key];
            }
          }
        }
        return target;
      }

	  function isLight(color) {
		let hex = color.replace('#', '');

		// convert #fff => #ffffff
		if(hex.length == 3){
			hex = `${hex[0]}${hex[0]}${hex[1]}${hex[1]}${hex[2]}${hex[2]}`;
		}

		const c_r = parseInt(hex.substr(0, 2), 16);
		const c_g = parseInt(hex.substr(2, 2), 16);
		const c_b = parseInt(hex.substr(4, 2), 16);
		const brightness = ((c_r * 299) + (c_g * 587) + (c_b * 114)) / 1000;
		return brightness > 155;
	}

	var bgColor = getComputedStyle(document.documentElement).getPropertyValue('--r-background-color').trim();
	var isLight = isLight(bgColor);

	if(isLight){
		document.body.classList.add('has-light-background');
	} else {
		document.body.classList.add('has-dark-background');
	}

      // default options to init reveal.js
      var defaultOptions = {
        controls: true,
        progress: true,
        history: true,
        center: true,
        transition: 'default', // none/fade/slide/convex/concave/zoom
        plugins: [
          RevealMarkdown,
          RevealHighlight,
          RevealZoom,
          RevealNotes,
          RevealMath.MathJax3,
		  RevealMermaid,
		  RevealChart,
		  RevealCustomControls,
	      RevealPointer,
		  RevealChalkboard, 
        ],


    	allottedTime: 120 * 1000,

		mathjax3: {
			mathjax: 'plugin/math/mathjax/tex-mml-chtml.js',
		},
		markdown: {
		  gfm: true,
		  mangle: true,
		  pedantic: false,
		  smartLists: false,
		  smartypants: false,
		},

		mermaid: {
			theme: isLight ? 'default' : 'dark',
		},

		customcontrols: {
			controls: [
				{ icon: '<i class="fa fa-pen-square"></i>',
				title: 'Toggle chalkboard (B)',
				action: 'RevealChalkboard.toggleChalkboard();'
				},
				{ icon: '<i class="fa fa-pen"></i>',
				title: 'Toggle notes canvas (C)',
				action: 'RevealChalkboard.toggleNotesCanvas();'
				},
			]
		},
      };

      // options from URL query string
      var queryOptions = Reveal().getQueryHash() || {};

      var options = extend(defaultOptions, {"width":960,"height":700,"margin":0.04,"controls":true,"progress":true,"slideNumber":false,"transition":"slide","transitionSpeed":"default"}, queryOptions);
    </script>

    <script>
      Reveal.initialize(options);
    </script>
  </body>

  <!-- created with Advanced Slides -->
</html>
